app/models/framework/definition/RM858.fdl

```diff
@@ -0,0 +1,66 @@
+Framework RM858 {
+  Name 'Pan Govt Vehicle Leasing & Fleet Outsource Solutio'
+
+  ManagementCharge varies_by 'Spend Code' {
+    'Lease Rental'         -> 0.5%
+    'Fleet Management Fee' -> 0.5%
+    'Damage'               -> 0%
+    'Other Re-charges'     -> 0%
+  }
+
+  InvoiceFields {
+    LotNumber from 'Lot Number'
+    optional CustomerPostCode from 'Customer PostCode'
+    CustomerName from 'Customer Organisation'
+    CustomerURN from 'Customer URN'
+    optional InvoiceDate from 'Customer Invoice Date'
+    InvoiceNumber from 'Customer Invoice Number'
+    optional String from 'Customer Invoice Line Number'
+    optional ProductDescription from 'Invoice Line Product / Service Description'
+    UnitType from 'Unit of Purchase'
+    optional UnitPrice from 'Price per Unit ex VAT'
+    optional UnitQuantity from 'Invoice Line Quantity'
+    InvoiceValue from 'Invoice Line Total Value ex VAT'
+    VATIncluded from 'VAT Applicable'
+    optional VATCharged from 'VAT Amount Charged'
+    PromotionCode from 'Spend Code'
+    ProductGroup from 'Invoice Line Product / Service Grouping'
+    optional ProductCode from 'CAP Code'
+    optional ProductClass from 'Vehicle Make'
+    optional ProductSubClass from 'Vehicle Model'
+    optional String Additional1 from 'Vehicle Derivative'
+    optional String Additional2 from 'Product Classification'
+    optional String Additional3 from 'Vehicle Registration'
+    optional String Additional4 from 'Vehicle Convertors Name'
+    optional String Additional5 from 'Vehicle Conversion Type'
+    optional String Additional6 from 'Vehicle Type'
+    optional String Additional7 from 'Fuel Type'
+    optional String Additional8 from 'CO2 Emission Levels'
+    optional String Additional9 from 'Lease Period'
+    Date Additional10 from 'Lease Start Date'
+    Date Additional11 from 'Lease End Date'
+    optional String Additional12 from 'Payment Profile'
+    optional Decimal Additional13 from 'Annual Lease Mileage'
+    optional Decimal Additional14 from 'Base Vehicle Price ex VAT'
+    optional Decimal Additional15 from 'Lease Cost excluding Optional Extras and Conversion ex VAT'
+    optional Decimal Additional16 from 'Lease Finance Charge ex VAT'
+    optional Decimal Additional17 from 'Lease Finance Margin ex VAT'
+    optional String Additional18 from 'Vehicle Purchase Terms'
+    optional Decimal Additional19 from 'Standard Vehicle Discount (%)'
+    optional Decimal Additional20 from 'Enhanced Vehicle Discount (%)'
+    optional Decimal Additional21 from 'Annual Service Maintenance & Repair Costs ex VAT'
+    optional Decimal Additional22 from 'Annual Breakdown & Recovery Costs ex VAT'
+    optional Decimal Additional23 from 'Residual Value'
+    optional String from 'Cost Centre'
+    optional String from 'Contract Number'
+  }
+
+  Lookups {
```

Comment:
Does 'Lookups' here mean essentially validates inclusion? That would be my assumption looking at this language from the first look, but I wouldn't be certain from this name. I'd lean towards something more explicit. Or maybe I'm misunderstanding something.

Reply:
It does, but we shouldn't use our implementation language here. It's a vanilla name at the moment; we're hoping that either of [dependent field validations](https://trello.com/c/FFJrLQDZ/899-dependent-field-lookups-rm6060) or further testing with Dave will manage to make this into something a bit more useful from a domain language perspective.